Tasks/AzureCLI/task.json

```diff
@@ -33,6 +33,7 @@
     "inputs": [
         {
             "name": "connectedServiceNameARM",
+            "aliases": ["azureSubscription"],
```

Comment:
Same comment as before.
Should we make aliases to be upperCamelCase / LowerCamelCase based on the other input case?

Reply:
Same as above.